README.md

```diff
@@ -21,7 +21,7 @@ Also see [TESTING.md](./TESTING.md) for details about automated testing.
 ### Windows
 
 1. Fork zetkin/app.zetking.org into your personal GitHub account.
-1. Install Git for Windows from https://git-scm.com/download/win
+1. Install Git from https://git-scm.com/download
```

Comment:
This is below the "Windows" headline, and the instructions in this list (as a whole) are specific to Windows, even if some of them are similar or identical to what one would do on other operating systems as well.